core/src/test/java/org/helsinki/paytrail/request/payments/PaytrailPaymentCreateTest.java

```diff
@@ -82,20 +111,19 @@ void createTestPayment() throws ExecutionException, InterruptedException {
         payload.setCallbackUrls(callbackUrls);
         payload.setRedirectUrls(callbackUrls);
 
-//        PaytrailPaymentCreateRequest request = new PaytrailPaymentCreateRequest(payload);
-//
-//        //  TODO Change to createResponse
-//        CompletableFuture<PaytrailResponse> response = client.sendRequest(request);
-//
-//        PaytrailResponse methodsResponse = mapper.to(response.get());
-//
-//
-//        try {
-//            String json = gson.toJson(response.get());
-//            System.out.println(json);
-//        } catch (InterruptedException | ExecutionException e) {
-//            throw new RuntimeException(e);
-//        }
+        PaytrailPaymentCreateRequest request = new PaytrailPaymentCreateRequest(payload);
+        CompletableFuture<PaytrailPaymentCreateResponse> response = client.sendRequest(request);
+
+        PaytrailPaymentCreateResponse paymentCreateResponse = mapper.to(response.get());
+
```

Comment:
Could you add some assertions to returning json? Like the values you give are returned in response?

Reply:
Sure!